users/serializers.py

```diff
@@ -32,12 +32,33 @@ class CustomerAccountSerializer(serializers.ModelSerializer):
     class Meta:
         model = CustomerAccount
         fields = [
-            'id', 'full_name',
+            'id', 'full_name', 'email',
             'phone', 'user_name', 'is_active', 'address', 'email', 'password']
         extra_kwargs = {
             'password': {'write_only': True, 'required': True}
         }
 
 
+class ResetPasswordRequestSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = PasswordReset
+        fields = ['email']
+
+
+class ResetPasswordSerializer(serializers.Serializer):
+    new_password = serializers.RegexField(
+        regex=r'^(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$',
+        write_only=True,
+        error_messages={
+            'invalid': (
+                'Password must be at least 8 characters long with '
+                'at least one capital letter and symbol'
+            )
+        }
+    )
+    confirm_password = serializers.CharField(write_only=True, required=True)
```

Comment:
What about the user who submits the password that is not the same as the confirmed password?